app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Database/Workout/WorkoutRepository.java

```diff
@@ -5,44 +5,61 @@
 import android.os.AsyncTask;
 import android.util.Log;
 
+import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 
+import edu.drexel.lapcounter.lapcounter.backend.Database.Device.DeviceDao;
+import edu.drexel.lapcounter.lapcounter.backend.Database.LapCounterDatabase;
+
 public class WorkoutRepository {
 
     private WorkoutDao mWorkoutDao;
-    private LiveData<List<Workouts>> mAllWorkouts;
 
 
     WorkoutRepository(Application application) {
-        WorkoutDatabase db = WorkoutDatabase.getDatabase(application);
+        LapCounterDatabase db = LapCounterDatabase.getDatabase(application);
         mWorkoutDao = db.workoutDao();
-        mAllWorkouts = mWorkoutDao.getAllWorkouts();
-    }
-
-    LiveData<List<Workouts>> getAllWorkouts() {
-        return mAllWorkouts;
     }
 
-    public void insert(Workouts workout)
+    public ArrayList<Workout> getAllWorkouts()
```

Comment:
As a matter of style, this should return a `List<Workout>`, not an `ArrayList<Workout>`